src/index.js

```diff
@@ -1,6 +1,27 @@
-const express = require('express');
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import { config } from 'dotenv';
+import usersRoutes from './routes/user.routes.js';
+
+config();
 const app = express();
+const PORT = process.env.PORT || 5000;
+
+
+app.use(cors());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(express.json()); 
+
+
+/* ========== Start:: User api url ========= */ 
+  app.use('/api/v1/users', usersRoutes);
+/* ============== Start:: User api ========= */ 
```

Comment:
remove this and put it inside a router/index.ts file.
That fill will export one router which will be called here as. 
```suggestion
  app.use(router);
```